Essentials/src/com/earth2me/essentials/utils/FormatUtil.java

```diff
@@ -19,6 +20,8 @@
     private static final Pattern STRIP_ALL_PATTERN = Pattern.compile("\u00a7+([0-9a-fk-orA-FK-OR])");
     //Essentials '&' convention colour codes
     private static final Pattern REPLACE_ALL_PATTERN = Pattern.compile("(&)?&([0-9a-fk-orA-FK-OR])");
+
+    private static final Pattern REPLACE_ALL_RGB_PATTERN = Pattern.compile("(&)?&#([A-Fa-f0-9]{6})");
```

Comment:
Just for code consistency
```suggestion
    private static final Pattern REPLACE_ALL_RGB_PATTERN = Pattern.compile("(&)?&#([0-9a-fA-F]{6})");
```